### DISCUSSION TITLE
Fossify Calculator will be re-released everywhere

### DISCUSSION THREAD

**Discussion by naveensingh** [FossifyOrg/General-Discussion#600] [Announcements]
The Fossify Calculator app has been [suspended](https://github.com/orgs/FossifyOrg/discussions/57#discussioncomment-11059627) from the Google Play Store. Unfortunately, the only way to resolve this issue is to re-release the app with a new application ID.

The app will be re-released on _both_ Google Play Store and FDroid to allow easy migration between Play/FDroid versions and to avoid user confusion.

**Comment by Kreuger** [comment]
Thank you for your tireless efforts. It is greatly appreciated.

**Comment by IzzySoft** [comment]
Please let me know if/when any action is needed from us for IzzyOnDroid, @naveensingh – and yeah, joining the Thank-you-crew of course! Bringing FOSS apps to PlayStore these days almost looks like too much hazzle (you're not the first dev I hear from with the issues listed behind the link; things like "provide credentials" for an app that does have none, etc)…

**Reply by naveensingh** [reply]
Will do, thanks!

**Reply by naveensingh** [reply]
It's up now: https://github.com/FossifyOrg/Calculator/releases/tag/1.0.3

@IzzySoft FDroid MR is already merged, you are gonna miss the train ;)

**Reply by IzzySoft** [reply]
Nope, won't. Will go live with the next sync around 7 pm UTC – and is RB :partying_face:

**Reply by IzzySoft** [reply]
Btw, what's with the old one, `org.fossify.calculator` – shall we remove that after an overlap?

![image](https://github.com/user-attachments/assets/0085991c-7531-4d19-b520-450ebaf53377)

We could locally add a line to its changelog or description that it's continued with the new package name (to give people a chance to get aware), leave it up for another month, and then remove it. Would that be in your interest?

**Reply by naveensingh** [reply]
Yes, we can remove it after an overlap or now but please choose whichever suits you best. There aren't many Fossify Calculator users (yet).

**Reply by IzzySoft** [reply]
OK, added a note to the en-US fulldesc of org.fossify.calculator, removing all other fulldesc so it's visible in all languages, then marked it for removal on/after 2025-04-20.
